guides/v3.5.0/tutorial/service.md

```diff
@@ -167,38 +123,30 @@ ember g component location-map
 
 Running this command generates three files: a component JavaScript file, a template, and a test file.
 
-Let's start by adding a `div` element to the component template.
-This `div` will act as a place for the 3rd party map API to render the map to.
-
-```handlebars {data-filename="app/templates/components/location-map.hbs" data-diff="-1,+2"}
-{{yield}}
-<div class="map-container"></div>
```

Comment:
since this div was only to add a class name and append the map, I figured I could just do this to the component top level element instead.